elifepubmed/generate.py

```diff
@@ -364,6 +365,17 @@ def set_abstract(parent, poa_article, abstract_label_types):
         set_abstract_text(abstract_tag, '', '')
 
 
+def set_plain_language_summary(parent, poa_article):
```

Comment:
do we always do POAs now? Wonder what happens if an article starts from a v1 VOR and if this is correctly named to cover that case?

Reply:
The named `poa_article` is a historically old name I never bothered to change, which is used often in this library and the Crossref generation library. Clearly the name is confusing. I could use `article` or `article_object` here instead, or an alternate name. Any preference? The chosen name could, in future, be used to rename all the `poa_article` names in all these libraries.

Reply:
Just `article`? About the scope of the refactoring, I'll leave that to you.

Reply:
👍